pkgs/misc/drivers/openzwave/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchFromGitHub
+, doxygen, libxml2, git, which, pkgconfig, graphviz-nox
+, udev }:
+
+let
+  version = "2018-04-04";
```

Comment:
Why not a stable version?

Reply:
Because the latest stable is V1.5 and was relaseaed 2016-08-15. And the repository seems to be quite active since then.

https://github.com/OpenZWave/open-zwave/releases/tag/V1.5

Reply:
Seems like almost all distros package 1.5: https://repology.org/metapackage/openzwave/versions.
But `pip install python_openzwave` installs from master by default: https://github.com/OpenZWave/python-openzwave#python-openzwave-04x-is-here-.
So I guess using an unstable version should be fine.
You could however ask them what they want distributions to do on their mailing list.